### PR TITLE
FOGL-7073: fixed code for unDeprecated StorageAssetTracking tuple

### PR DESCRIPTION
**Pull request by Innovative-ashwin**
Signed-off-by: Innovative-ashwin <sinha.ashwini111@gmail.com>

### CODE REVIEW

**Review by MarkRiddoch — APPROVED** [review]

File: C/services/south/ingest.cpp [hunk 1]
```diff
@@ -1206,11 +1206,27 @@ void Ingest::unDeprecateStorageAssetTrackingRecord(StorageAssetTrackingTuple* cu
...
+        string temp;
+        while(getline(dpStringStream, temp, ','))
+        {
+	        tokens.push_back(temp);
+        }
+
+	ostringstream convert;
+        convert << "{";
+        convert << "\"datapoints\":[";
+        for (unsigned int i = 0; i < tokens.size() ; ++i)
```
Comment (AmandeepArora) [inline C/services/south/ingest.cpp:1220]: There is probably no need to store tokens temporarily, you can tokenize and use tokens immediately.

**Review by AmandeepArora — APPROVED** [review]

**Review by deepanshu-yadav — APPROVED** [review]
Solved 7076 second issue.

**Review by praveen-garg — APPROVED** [review]
lgtm!

File: C/services/south/ingest.cpp (hunk 1 above)
Comment (Innovative-ashwin) [reply C/services/south/ingest.cpp:1220]: minor

**Pull request merged**